daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -16,7 +16,7 @@ This table is meant to help users understand the equivalent options for running
 | `--app-id` | `--app-id` | `-i` | `dapr.io/app-id`  | The unique ID of the application. Used for service discovery, state encapsulation and the pub/sub consumer ID |
 | `--app-port` | `--app-port` | `-p` | `dapr.io/app-port` | This parameter tells Dapr which port your application is listening on |
 | `--app-ssl` | `--app-ssl` | | `dapr.io/app-ssl` | Sets the URI scheme of the app to https and attempts an SSL connection |
-| `--components-path`  | `--components-path` | `-d` | not supported | Path for components directory. If empty, components will not be loaded. |
+| `--resources-path`  | `--resources-path` | `-d` | not supported | Path for components directory. If empty, components will not be loaded. |
```

Comment:
@pravinpushkar - Can you clarify here that components-path is not removed in this release, it is just marked as deprecated. In otherwords should we keep both here for now with a note to say components-path is deprecated?

Reply:
same as above

Reply:
Agreed.

Can we have both and just mention that components-path is deprecated?

Reply:
@mukundansundar I've updated per these comments!